### ISSUE TITLE
Dang! Nice Brian!

### ISSUE THREAD

**Issue by ellismarte** [ellismarte/FACQ#4]
This looks great.

**Comment by briankennedy1** [comment]
Yeah, I think I'll tweet these out and probably add them as blog posts as well. Super simple things that just don't seem to come up in search engines for us beginners.

**Comment by ellismarte** [comment]
I couldn't agree more. The lack of material showing up on search engines
that is relevant to us beginners is what makes me want to give back.
